src/aggregate/aggregate_request.c

```diff
@@ -461,6 +453,7 @@ int PLNGroupStep_AddReducer(PLN_GroupStep *gstp, const char *name, ArgsCursor *a
   if (AC_AdvanceIfMatch(ac, "AS")) {
     rv = AC_GetString(ac, &alias, NULL, 0);
     if (rv != AC_OK) {
+      array_pop(gstp->reducers);
```

Comment:
should check null before freeing instead

Reply:
same comment